plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx

```diff
@@ -95,11 +95,11 @@ export const UserProfileCard = ({
   return (
     <InfoCard title={<CardTitle title={displayName} />} variant={variant}>
       <Grid container spacing={3} alignItems="flex-start">
-        <Grid item md={2} lg={1}>
+        <Grid item xs={12} sm={2} xl={1}>
           <Avatar displayName={displayName} picture={profile?.picture} />
         </Grid>
 
-        <Grid item md={10} lg={11}>
+        <Grid item md={10} xl={11}>
```

Comment:
odd that these don't line up with the breakpoints above - would have expected something like `xs={12} sm={10} xl={1}`

Reply:
Hmm, I'll take another look. Visually they looked fine but perhaps I missed something.